specifications/17-dependent-features.json

```diff
@@ -0,0 +1,885 @@
+            "stickiness": "default",
+            "groupId": "groupId"
+          },
+          "variants": [
+            {
+              "name": "child.variant",
+              "weight": 1,
+              "payload": {
+                "type": "string",
+                "value": "variantValue"
+              }
+            }
+          ]
+        }]
+      },
+      {
+        "name": "parent.disabled.not.satisfied.child.enabled",
+        "description": "Parent disabled expectation not satisfied, child enabled",
+        "enabled": true,
+        "dependencies": [{
+          "feature": "parent.enabled",
+          "enabled": false
+        }],
+        "strategies": [{
+          "name": "flexibleRollout",
+          "parameters": {
+            "rollout": "100",
+            "stickiness": "default",
+            "groupId": "groupId"
+          },
+          "variants": [
+            {
+              "name": "child.variant",
+              "weight": 1,
+              "payload": {
+                "type": "string",
+                "value": "variantValue"
+              }
+            }
+          ]
+        }]
+      },
+      {
+        "name": "parent.single.variant.child.enabled",
+        "description": "Parent single variant match, child enabled",
+        "enabled": true,
+        "dependencies": [{
+          "feature": "parent.with.variant",
+          "variants": ["parent.variant"]
+        }],
+        "strategies": [{
+          "name": "flexibleRollout",
+          "parameters": {
+            "rollout": "100",
+            "stickiness": "default",
+            "groupId": "groupId"
+          },
+          "variants": [
+            {
+              "name": "child.variant",
+              "weight": 1,
+              "payload": {
+                "type": "string",
+                "value": "variantValue"
+              }
+            }
+          ]
+        }]
+      },
+      {
+        "name": "parent.multiple.variants.child.enabled",
+        "description": "Parent multiples variants match, child enabled",
+        "enabled": true,
+        "dependencies": [{
+          "feature": "parent.with.variant",
+          "variants": ["parent.variant", "nonmatching.variant"]
+        }],
+        "strategies": [{
+          "name": "flexibleRollout",
+          "parameters": {
+            "rollout": "100",
+            "stickiness": "default",
+            "groupId": "groupId"
+          },
+          "variants": [
+            {
+              "name": "child.variant",
+              "weight": 1,
+              "payload": {
+                "type": "string",
+                "value": "variantValue"
+              }
+            }
+          ]
+        }]
+      },
+      {
+        "name": "parent.empty.variants.child.enabled",
+        "description": "Parent empty variants match, child enabled",
+        "enabled": true,
+        "dependencies": [{
+          "feature": "parent.with.variant",
+          "variants": []
```

Comment:
This operates on `results` I take it? Do default-variants count as []?

Reply:
Ah, great! So `"variants": [ ]` is the same as not including the variants property in the dependency